gwy/templates/Dockerfile.tmpl

```diff
@@ -1,7 +1,8 @@
 # Template for grpc-gateway 
 
-ARG alpine_version=3.8
-FROM golang:1.10-alpine$alpine_version AS build
+ARG alpine=3.11
+ARG go=1.12
```

Comment:
The current stable version is `1.14`. Why `1.12`?

Reply:
I'm matching what is already used in the other images in this repo. My goal is to get everything on the same version. We can update them all together in the (hopefully near) future.

Reply:
OK. Sorry.